pkg/types/types.go

```diff
@@ -152,8 +152,8 @@ const (
 )
 
 type Metrics struct {
-	Bandwidth    RWMetrics // in byte
-	TotalLatency RWMetrics // in microsecond(us)
+	Throughput   RWMetrics // in byte
+	TotalLatency RWMetrics // in nanoseconds
```

Comment:
Using a more accurate naming and a finer unit is good. But will these lead to compatibility issues?

Reply:
I've checked the MetricGet method. It's not integrated into longhorn-manager, so there is not compatibility issue, right?